core/src/components/slider/slider.spec.ts

```diff
@@ -66,25 +68,31 @@ describe(`Slider basic`, () => {
 		const sliderDirective = slider.directives.sliderDirective(sliderElement);
 		const minLabelDirective = slider.directives.minLabelDirective(minLabel);
 		const maxLabelDirective = slider.directives.maxLabelDirective(maxLabel);
-		const unsubscribe = slider.state$.subscribe((newState) => {
-			state = newState;
+
+		normalizedState$ = computed(() => {
```

Comment:
mb we should extract the method to get the normalize state in case we have some other fields?